ethereum/contracts/Pausable.sol

```diff
@@ -0,0 +1,32 @@
+pragma solidity 0.5.10;
+
+import './Ownable.sol';
+
+contract Pausable is Ownable {
+  bool private _paused = false;
```

Comment:
haha ... like that. Should add a getter so it's easy to inspect the pause/run state.